src/Shout-Tests/SHRBMockSharedPoolStyler.class.st

```diff
@@ -0,0 +1,11 @@
+"
+I am a shared pool used for styler tests.
+"
+Class {
+	#name : #SHRBMockSharedPoolStyler,
```

Comment:
I do not see where it is used

Reply:
It is used as a shared pool in the test class to be able to test shared pool style

Reply:
![0FC59509-323C-4181-A181-FB79A0D451D6](https://user-images.githubusercontent.com/9519971/223670113-674d10fd-52b5-46ae-a4a1-79aa3587b18e.jpeg)